jni/CHANGELOG.md

```diff
@@ -1,3 +1,20 @@
+## 0.8.0-wip
+
+- **Breaking Change** ([#394](https://github.com/dart-lang/jnigen/issues/394)):
+  Converted various `Exception`s into `Error`s:
+  - `UseAfterReleaseException` -> `UseAfterReleaseError`
+  - `DoubleReleaseException` -> `DoubleReleaseError`
+  - `SpawnException` -> `JniError` (It's now a `sealed class`)
+  - `JNullException` -> `JNullError`
+  - `InvalidCallTypeException` -> `InvalidCallTypeError`
+  - `HelperNotFoundException` -> `HelperNotFoundError`
+  - `JvmExistsException` -> `JniVmExistsError`
+  - `NoJvmInstanceException` -> `NoJvmInstanceError`
+- **Breaking Change**: Removed `InvalidJStringException`.
+- **Breaking Change**: The default return `callType` of `int` is Java's `long`.
+
+[TODO(#393): Add the fact that `JException` is now a `JObject`.]: #
```

Comment:
This is invalid markdown right? It looks like it should match `TODO(#393): Add the fact that `JException` is now a `JObject`.` earlier in the text and make a link to `#`.

Reply:
It's a weird way to make a comment in a markdown (so it won't get shown in the preview).

Reply:
Sure!